unstructured/ingest/pipeline/utils.py

```diff
@@ -1,8 +1,7 @@
 import hashlib
-import json
 
 
-def get_ingest_doc_hash(doc: str) -> str:
-    json_as_dict = json.loads(doc)
-    hashed = hashlib.sha256(json_as_dict.get("filename").encode()).hexdigest()[:32]
+def get_ingest_doc_hash(json_as_dict: dict) -> str:
+    print(json_as_dict.keys())
```

Comment:
do we want this print here? or was this for debugging?

Reply:
nope, this was from a previous update I was testing. Not needed anymore.